src/components/thumbnail/class-logs.tsx

```diff
@@ -18,7 +18,7 @@ export class ClassLogsComponent extends BaseComponent<IProps, {}> {
     const publications = documents.getLatestLogPublications();
 
     return (
-      <div className="class-work">
+      <div className="class-logs">
```

Comment:
Good catch. Worth checking if there are any CSS implications here.

Reply:
None at present that I could find, but it looked like something that might bite us later.